### DISCUSSION TITLE
Recommend sizing of CPU/Memory needed for docker containers for self hosting?

### DISCUSSION THREAD

**Discussion by nagakirankys** [umami-software/umami#2715] [General]
I am looking at recommend sizing of CPU/Memory to be configured for docker containers for self hosting but couldn't find any reference in documentation/issues in this repo.

Can someone please suggest the recommend sizing of CPU/Memory for given number of events/day?

**Comment by FHU-yezi** [comment]
For my deployment use the PostgreSQL version, Umami use about 100MB, and I limit the memory usage to 256MB.

I have about 500 events per day.

**Comment by YIOrainy** [comment]
Did you find anything ?
i am also searching
